### PR TITLE
add: support check pulsar partition topic metadata

### PR DESCRIPTION
**Pull request by xuthus5**

### CODE REVIEW

File: src/main/java/com/github/shoothzj/zdash/controller/ZnodeController.java [hunk 1]
```diff
@@ -151,4 +158,85 @@ public ResponseEntity<SupportDecodeNamespaceListResp> getDecodeNamespaces() {
         return new ResponseEntity<>(namespaceListResp, HttpStatus.OK);
     }
 
+    @PostMapping("/check-pulsar-partition-topic-metadata")
+    public ResponseEntity<DiffPartitionResp> checkPulsarPartitionTopicMetadata() throws Exception {
+        ZooKeeper zooKeeper;
+        try {
+            zooKeeper = zkService.newZookeeper();
+        } catch (IOException e) {
+            log.error("create new zookeeper connect failed: {}", e);
```
Comment (Shoothzj) [inline src/main/java/com/github/shoothzj/zdash/controller/ZnodeController.java:167]: we can directly throw this exception, the framework will catch this

File: src/main/java/com/github/shoothzj/zdash/controller/ZnodeController.java [hunk 2]
```diff
@@ -151,4 +157,79 @@ public ResponseEntity<SupportDecodeNamespaceListResp> getDecodeNamespaces() {
         return new ResponseEntity<>(namespaceListResp, HttpStatus.OK);
     }
 
+    @PostMapping("/check-pulsar-partition-topic-metadata")
+    public ResponseEntity<DiffPartitionResp> checkPulsarPartitionTopicMetadata() throws Exception {
+        ZooKeeper zooKeeper = zkService.newZookeeper();
```
Comment (Shoothzj) [inline src/main/java/com/github/shoothzj/zdash/controller/ZnodeController.java:162]:
```suggestion
        try(ZooKeeper zooKeeper = zkService.newZookeeper())
```

File: src/main/java/com/github/shoothzj/zdash/module/pulsar/DiffPartitionResp.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+import java.util.List;
+
+@Setter
+@Getter
+public class DiffPartitionResp {
+
+    @Setter
+    @Getter
+    public static class DiffPartition {
+        private String topic;
```
Comment (Shoothzj) [inline src/main/java/com/github/shoothzj/zdash/module/pulsar/DiffPartitionResp.java:34]: only `topic`? how about `topic`、`partition`

File: src/main/java/com/github/shoothzj/zdash/module/pulsar/PartitionInfo.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+package com.github.shoothzj.zdash.module.pulsar;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Setter
+@Getter
+public class PartitionInfo {
+    private Integer partitions;
```
Comment (Shoothzj) [inline src/main/java/com/github/shoothzj/zdash/module/pulsar/PartitionInfo.java:28]:
```suggestion
    private int partitions;
```

File: src/main/java/com/github/shoothzj/zdash/module/pulsar/DiffPartitionResp.java (hunk 3 above)
Comment (xuthus5) [reply src/main/java/com/github/shoothzj/zdash/module/pulsar/DiffPartitionResp.java:34]: accept

File: src/main/java/com/github/shoothzj/zdash/module/pulsar/PartitionInfo.java (hunk 4 above)
Comment (xuthus5) [reply src/main/java/com/github/shoothzj/zdash/module/pulsar/PartitionInfo.java]: accept

**Review by Shoothzj — APPROVED** [review]

**Pull request merged**